src/app/base/components/PowerTypeFields/LXDPowerFields/LXDPowerFields.test.tsx

```diff
@@ -21,9 +21,9 @@ describe("LXDPowerFields", () => {
   });
 
   it("can be given a custom power parameters name", () => {
-    const field = powerFieldFactory({ name: "field" });
+    const field = powerFieldFactory({ name: "field", label: "custom field" });
     const store = mockStore(state);
-    const wrapper = mount(
+    render(
```

Comment:
This and the other tests in this file should be able to use `renderWithMockStore`